ksql-engine/src/main/java/io/confluent/ksql/planner/plan/StructuredDataSourceNode.java

```diff
@@ -84,6 +86,9 @@
         return row;
       };
 
+  private static final String SOURCE_OP_NAME = "source";
+  private static final String REDUCE_OP_NAME = "reduce";
```

Comment:
Same comment as above: the name `REDUCE_OP_NAME` is somewhat confusing in light of `reduceOpName()` below.